### DISCUSSION TITLE
Unable to change lock dir

### DISCUSSION THREAD

**Discussion by peppepetra** [strongswan/strongswan#1542] [General]
When building strongSwan I am not able to change the dir `/var/lock/subsys`. 

Is there a parameter that I can pass to configure to change that?

I am currently using `--with-piddir` which changes the directory of `/var/run` but I am not able to change the one for `/var/lock`

Thanks.

**Comment by tobiasbrunner** [comment]
There is no parameter, so you have to change the legacy `ipsec` script manually (or the `_ipsec.in` source file). However, if the directory doesn't exist, the script just ignores it. And if it does, the file it creates there with `ipsec start` and removes with `ipsec stop` might not be necessary anyway as the script itself does not check for it. It was added with 7ebac6728bb0f6025ec8c7980d2a646cdcb79f86, but that doesn't really explain why it was added to the `ipsec` script and not e.g. to the actual init script that calls it (I don't think the `ipsec` script was ever intended as actual init script - but maybe the Fedora package did at the time).
